docs/code-examples/depth_image_3d.py

```diff
@@ -9,7 +9,7 @@
 image[130:180, 100:280] = 45000
 
 
-rr.init("depth_image", spawn=True)
+rr.init("rerun-example-depth_image", spawn=True)
```

Comment:
I think all these actually look quite ugly. I'd honestly prefer if we just used regular whitespace for all the recording id's. What's even the reason for using snake or kebab case?

Reply:
With "these" I'm particularly referring to the snake_-kebab

Reply:
The reason is that this is an _id_, not a name.

A user reading `"Rerun example: annotation context rects"` is likely to interpret that string as a title, and would probably be surprised that changing it slightly would have side-effects, like Rerun no longer associating a previously stored blueprint with the new recording.